src/test/suite/index.ts

```diff
@@ -3,6 +3,17 @@ import * as Mocha from 'mocha';
 import * as glob from 'glob';
 
 export function run(): Promise<void> {
+    if (process.env.PATH && process.env.RUBY_BIN_PATH) {
+        console.log(
+            "Prefer a newly-installed Ruby version over the system's default one." +
+                ' This is a workaround for https://github.com/microsoft/vscode-test/issues/20',
+        );
+        const previous = process.env.PATH;
+        process.env.PATH = [process.env.RUBY_BIN_PATH, ...previous.split(path.delimiter)].join(path.delimiter);
+        console.log(`Previous PATH: ${previous}`);
+        console.log(` Current PATH: ${process.env.PATH}`);
```

Comment:
nit
```suggestion
        console.log(`Current PATH: ${process.env.PATH}`);
```

Reply:
I think this is matching the columns of the colon. I am fine with the original code.